.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+GitHub Issue (If applicable): #
+<!-- Link to relevant GitHub issue if applicable. All PRs should be associated with an issue (GitHub issue or internal) -->
+
+## PR Type
+What kind of change does this PR introduce?
+<!-- Please uncomment one ore more that apply to this PR -->
+
+<!-- - Bugfix -->
+<!-- - Feature -->
+<!-- - Code style update (formatting) -->
+<!-- - Refactoring (no functional changes, no api changes) -->
+<!-- - Build or CI related changes -->
+<!-- - Documentation content changes -->
+<!-- - Other... Please describe: -->
+
+
+## What is the current behavior?
+<!-- Please describe the current behavior that you are modifying, or link to a relevant issue. -->
+
+
+## What is the new behavior?
+<!-- Please describe the new behavior after your modifications. -->
+
+
+## PR Checklist
+
+Please check if your PR fulfills the following requirements:
+
+- [ ] Tested code with current [supported SDKs](../README.md#supported)
+- [ ] Docs have been added/updated which fit [documentation template](https://github.com/nventive/Uno/blob/master/doc/.feature-template.md). (for bug fixes / features)
```

Comment:
docs here point to uno. I think we can remove this checkbox line altogether, unless we have the equivalent in uno.core